src/AbstractPolytope.jl

```diff
@@ -198,5 +205,12 @@ end
 function default_polyhedra_backend(P, N::Type{<:Rational})
     return default_library(LazySets.dim(P), Rational{Int})
 end
+
+import JuMP, GLPK
```

Comment:
I would add these `import`s to the top of the `load_polyhedra_abstractpolytope` block.